libnetwork/ipams/defaultipam/allocator.go

```diff
@@ -215,6 +216,7 @@ func newPoolData(pool netip.Prefix) *PoolData {
 
 	// Allow /64 subnet
 	if pool.Addr().Is6() && numAddresses == 0 {
+		// FIXME(robmry) - see the comment in getAddress
 		numAddresses--
 	}
```

Comment:
OHMAN yeah, this is confusing `numAddresses == 0` as the condition .. so let's decrement :confused_travolta: